ChatSecure/Classes/View Controllers/Login View Controllers/OTRAccountMigrationViewController.m

```diff
@@ -116,6 +117,11 @@ - (void) handleSuccessWithNewAccount:(OTRXMPPAccount*)newAccount sender:(id)send
     
     XMPPvCardTemp *vCard = self.oldAccount.vCardTemp;
     vCard.jid = newAccount.bareJID;
+    self.oldAccount.waitingForvCardTempFetch = NO;
+    self.oldAccount.lastUpdatedvCardTemp = [NSDate date];
+    [[OTRDatabaseManager sharedInstance].readWriteDatabaseConnection readWriteWithBlock:^(YapDatabaseReadWriteTransaction * _Nonnull transaction) {
```

Comment:
@N-Pex For future reference there's a new singleton accessor that's a little cleaner:

```objc
OTRDatabaseManager.shared.readWriteDatabaseConnection
```